pallene/coder.lua

```diff
@@ -1437,6 +1437,42 @@ gen_cmd["BuiltinIoWrite"] = function(self, cmd, _func)
     return util.render([[ pallene_io_write(L, $v); ]], { v = v })
 end
 
+gen_cmd["BuiltinMathAbs"] = function(self, cmd, _func)
+    local dst = self:c_var(cmd.dsts[1])
+    local v = self:c_value(cmd.srcs[1])
+    return util.render([[ $dst = l_mathop(fabs)($v); ]], { dst = dst, v = v })
+end
+
+gen_cmd["BuiltinMathExp"] = function(self, cmd, _func)
+    local dst = self:c_var(cmd.dsts[1])
+    local v = self:c_value(cmd.srcs[1])
+    return util.render([[ $dst = l_mathop(exp)($v); ]], { dst = dst, v = v })
+end
+
+gen_cmd["BuiltinMathLog"] = function(self, cmd, _func)
+    local dst = self:c_var(cmd.dsts[1])
+    local v = self:c_value(cmd.srcs[1])
+    local b = self:c_value(cmd.srcs[2])
+	local base_num = tonumber(b)
```

Comment:
We can use spaces to indent this instead of tabs, like the rest of the codebase.

Reply:
Aha! I know that linter check would pay off some day. :)

Reply:
Oops. Sorry, I missed that one. I just fixed that line. How do I amend the PR with GitHub to include this change?

Reply:
I believe just pushing to this branch should automatically update the PR.